cmd/inspect/inspect_policy.go

```diff
@@ -119,10 +128,65 @@ func inspectPolicyCmd() *cobra.Command {
 	cmd.Flags().StringArrayVarP(&sourceUrls, "source", "s", []string{}, "policy source url. multiple values are allowed")
 	cmd.Flags().StringVarP(&destDir, "dest", "d", "", "use the specified destination directory to download the policy. if not set, a temporary directory will be used")
 	cmd.Flags().StringVarP(&outputFormat, "output", "o", "text", fmt.Sprintf("output format. one of: %s", strings.Join(validFormats, ", ")))
+	cmd.Flags().StringVar(&ruleFilter, "rule", ruleFilter, "display results matching rule name")
+	cmd.Flags().StringVar(&packageFilter, "package", packageFilter, "display results matching package name")
+	cmd.Flags().StringVar(&collectionFilter, "collection", collectionFilter, "display rules included in given collection")
 
 	if err := cmd.MarkFlagRequired("source"); err != nil {
 		panic(err)
 	}
 
 	return cmd
 }
+
+func filterResults(results map[string][]*ast.AnnotationsRef, rule, pkg, collection string) (map[string][]*ast.AnnotationsRef, error) {
+	if rule == "" && collection == "" {
```

Comment:
I guess `pkg` as well?

```suggestion
	if rule == "" && pkg == "" && collection == "" {
```

Reply:
Thank you!